tests/test_misc.py

```diff
@@ -51,6 +48,34 @@ def test_attrs_parse__filter():
     assert attrs_parse(lines, ["param2", "param3"]) == expected
 
 
+test_attrs_parse__filter()
+
+
+@pytest.mark.parametrize(
+    "input,expected",
+    [
+        (
+            ["cn: sudorules", "distinguishedName: objectSID=123,cn=id_m", " appings,cn=test,cn=sysdb"],
+            {"cn": ["sudorules"], "distinguishedName": ["objectSID=123,cn=id_mappings,cn=test,cn=sysdb"]},
+        ),
+        (
+            [
+                "description: My teacher is a goo",
+                " d one but I do not like him",
+                "  very much: he wears a dirty coat.",
```

Comment:
Can you update the string like this to cover trailing spaces?

```
            [
                "description: My teacher is a goo",
                " d one but I do not like him",
                "  very much: he wears a dirty ",
                " coat.",
            ],
```

Reply:
Yes, but this way test still passes, because in inner while there is no strip. I change it a bit more so it fails on this implementation.